data/mods/Magiclysm/Spells/druid.json

```
@@ -368,5 +368,33 @@
     "min_duration": 60000,
     "max_duration": 150000,
     "duration_increment": 6000
+  },
+  {
+    "id": "summon_wolf_druid",
+    "type": "SPELL",
+    "name": "Summon Wolf",
+    "description": "Call to the wilds for aid from some wolfs",
+    "valid_targets": [ "ground" ],
+    "min_damage": 1,
+    "max_damage": 3,
+    "damage_increment": 0.07,
```

```suggestion
    "damage_increment": 0.08,
```
So skill level 25 should get you 3 wolves (might have been doing it already due to rounding?).

---

Truth be told i'm not 100% sure how increments work :) but you seem to know more then me so i will make the commit